### ISSUE TITLE
It would be awesome if you could add a spot for apidoc calls

### ISSUE THREAD

**Issue by Jnorm911** [sphinx-notes/pages#16]
An option before the push that would allow us to enter a popular apidoc call for example "sphinx-apidoc -f -o . .." With this we could completely automate sphinx docs on push.

**Comment by SilverRainZ** [comment]
Hello, sorry for replying so late.

Do you have an example of this? If we generate apidoc for a python object, we do not need to call sphinx-apidoc by hand.

**Comment by hesic73** [comment]
I've encountered the same issue and have implemented some modifications in my fork available at https://github.com/hesic73/pages, where I also introduced a parameter `module_path`. To enable `sphinx-build` to locate the module, it's essential to add the following lines to conf.py:

```python
import os
import sys
sys.path.insert(0, os.path.abspath('..'))
```

**Comment by SilverRainZ** [comment]
Hi @hesic73, what the value of `module_path` should be?

B.T.W. Are you willing to make a pull request?

**Comment by hesic73** [comment]
According to https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html, MODULE_PATH is the path to a Python package to document. For example in [my repository](https://github.com/hesic73/gomoku_rl), `module_path=gomoku_rl`. 

For now it's hard-coded. But this should be an optional parameter. Maybe I will change the code and make a pr.

**Comment by SilverRainZ** [comment]
Yes, it would be better to make the parameter optional. Besides, I think `apidoc_module_path` should better than `module_path`.
